main.js

```diff
@@ -12,9 +12,46 @@ window.addEventListener('load', trigger.bind(null, 'load'));
 window.addEventListener('load', trigger.bind(null, 'DOMContentLoaded'));
 document.addEventListener('DOMContentLoaded', trigger.bind(null, 'DOMContentLoaded'));
 
+document.onreadystatechange = function () {
+	if (document.readyState === 'complete') {
+		trigger('DOMContentLoaded');
+		trigger('load');
+	} else if (document.readyState === 'interactive' && !document.attachEvent) {
+		trigger('DOMContentLoaded');
+	}
+};
+
 if (document.readyState === 'complete') {
-	trigger('load');
 	trigger('DOMContentLoaded');
-} else if (document.readyState === 'interactive') {
+	trigger('load');
+} else if (document.readyState === 'interactive' && !document.attachEvent) {
 	trigger('DOMContentLoaded');
 }
+
+
+if (document.attachEvent) {
+	// If IE and not a frame
+	// continually check to see if the document is ready
+	var top = false;
+
+	try {
+		top = window.frameElement == null && document.documentElement;
+	} catch(e) {}
+
+	if (top && top.doScroll) {
```

Comment:
It's native on document.documentElement:

![image](https://cloud.githubusercontent.com/assets/4048999/10017638/e73689fa-6138-11e5-95a1-3a3e05f8ba5b.png)